source/code-generation/query-converter-test-bench.txt

```diff
@@ -4,16 +4,56 @@
 Query Converter Test Bench
```

Comment:
The UI doesn't use this name; it just says "Test queries". I think it would be better to avoid the internal Test Bench name in the docs.

Reply:
I dont think the UI has any labels or titles at all TBH, I see `Test queries` to the left of the UI, but that dosent look like an official title (like the Query Converter pane for example). Would it be possible to add a UI element for the name of the feature? This would make documenting and referring to it easier. 

A few examples:
- Query Converter Query Tester
- Query Converter Results Tester
- Query Converter Tester
- Query Results Comparison

....

<img width="1623" alt="Screenshot 2024-07-29 at 2 33 21 PM" src="https://github.com/user-attachments/assets/40c0f6ea-0d1d-4bba-a9c9-311eb47ab03c">

Reply:
As discussed in the call - the labelling of the feature in the product was a conscious design decision. Given the pane is labelled "Test queries" I'd prefer to use the same name in the docs. If we need to nounify it we can call it the "Test queries pane".